include/dxc/CMakeLists.txt

```diff
@@ -10,13 +10,19 @@ configure_file(
   ${LLVM_INCLUDE_DIR}/dxc/Test/TestConfig.h
   )
 
+configure_file(
+  ${CMAKE_CURRENT_SOURCE_DIR}/config.h.cmake
+  ${LLVM_INCLUDE_DIR}/dxc/config.h
+  )
+
 add_subdirectory(DXIL)
 add_subdirectory(DxilContainer)
 add_subdirectory(HLSL)
 add_subdirectory(Support)
 add_subdirectory(Tracing)
 
 set(DXC_PUBLIC_HEADERS
+    ${CMAKE_CURRENT_BINARY_DIR}/config.h
```

Comment:
@hekota & @pow2clk, we need to make sure this header gets picked up in the DXC packages.

Reply:
Filed #5414 to track making sure the header gets into our releases.